src/main/java/milutrock/MiluTrock.java

```diff
@@ -17,7 +21,7 @@ public MiluTrock(String path) {
         storage = new Storage(path, parser);
     }
 
-    public void run() {
+    private void run() {
```

Comment:
The method `run` is concise and accurately describes what it does. However, since this method was changed to private, it might be beneficial to consider a more specific name that reflects its scope and usage, such as `executeTasks` or `startProcessing`.